### ISSUE TITLE
What happens if an application is opened with context?

### ISSUE THREAD

**Issue by rikoe** [FDC3/API#21]
It is unclear to me from the API documentation what happens if an application is launched with the `open(name: String, context: Context): Promise<void>` function.

My assumption is that the context will be handled by the context listener (`contextListener(handler: (context: Context) => void): Listener`), but what does this mean? Is opening an app with a context the same as first calling `open` without a context, followed by `broadcast` with a context?

If this is the case, do we really need the context as part of opening an application? Isn't it more explicit and less ambiguous for an application to call `broadcast` if it wants to send some context to an application.

I would appreciate further insight as to when it would make sense to send an arbitrary payload to an application on startup when it is not in the context of a particular intent.

I guess I don't mind this functionality, but I think the intended usage should be made clear in the documentation.

**Comment by nkolba** [comment]
Yes, the intention is that the context would go to the contextListener.  As you point out, this would be functionally equivalent to opening the app with no context and broadcasting the context directly to it.  Main value right now is convenience.

**Comment by nkolba** [comment]
To Do: update comments in the documentation to reflect the above clarification
